config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  	
+devise_for :users, :controllers => { registrations: 'registrations' }
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+   resources :posts
```

Comment:
- [ ] **flags:** you need to set up the right parameters in your resource, for this case will use the **`only`** or **`except`** flag
for example:
```ruby
# enable routing access to specifics methods
resources :posts, only: [:new, :create, :index]
```